### PR TITLE
docs(qa): advise to use temporary user accounts

### PR DESCRIPTION
**Pull request by kim**
For those unverbesserliche, we also provide a bash script to remove state and config data.

### CODE REVIEW

**Review by NunoAlexandre — DISMISSED** [review]
Sick :ok_hand:

File: QA.md [hunk 1]
```diff
@@ -4,8 +4,32 @@
...
+
+     **Dangerous method**: Remove all directories manually
+
+     You can use [this script](./scripts/reset-state.sh). Make sure you have a
+     backup of your data, or are using this in combination with the safe method
+     (i.e. while logged in with a temporary user account).
+
 - [ ] download the binary package from the location provided above
-- [ ] remove any old configuration and user data
+- [ ] install the binary package
```
Comment (kim) [inline QA.md:32]: Is this dance on macOS still necessary?

Comment (rudolfs) [reply QA.md:32]: Yeah, we should leave it in as it's not very obvious. Much macOS, such secure!

**Review by rudolfs — APPROVED** [review]
Very nice, I like! 🚀

**Review by NunoAlexandre — APPROVED** [review]

**Review by NunoAlexandre — APPROVED** [review]

**Pull request merged**